staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	namespace = "apiserver"
+	subsystem = "flowcontrol"
+)
+
+const (
+	priorityLevel = "priorityLevel"
+	flowSchema    = "flowSchema"
+)
+
+var (
+	queueLengthBuckets            = []float64{0, 10, 25, 50, 100, 250, 500, 1000}
+	requestDurationSecondsBuckets = []float64{0, 0.005, 0.02, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10, 30}
+)
+
+func init() {
+	prometheus.MustRegister(apiserver_rejected_requests)
+	prometheus.MustRegister(apiserver_current_inqueue_requests)
+	prometheus.MustRegister(apiserver_request_queue_length)
+	prometheus.MustRegister(apiserver_request_concurrency_limit)
+	prometheus.MustRegister(apiserver_current_executing_requests)
+	prometheus.MustRegister(apiserver_requests_waiting_seconds)
+	prometheus.MustRegister(apiserver_requests_execution_seconds)
+}
+
+var (
+	apiserver_rejected_requests = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "apiserver_rejected_requests",
+			Help:      "Number of rejected requests by request-management system",
```

Comment:
I suggest the more specific term "priority and fairness" rather than "request-management".

Reply:
we leave after agreeing on renaming the feature-gate

Reply:
The name of the feature gate does not need to dictate the name used everywhere.  It already does _not_.  The metric names do not use "request management".  The API group does not use "request management".  The documentation being drafted does not use "request management".

Reply:
renamed